core/src/Pos/Core/Common/AddrSpendingData.hs

```diff
@@ -123,3 +124,10 @@ addrSpendingDataToType =
         ScriptASD {} -> ATScript
         RedeemASD {} -> ATRedeem
         UnknownASD tag _ -> ATUnknown tag
+
+
+-- Define these at the end of the file to avoid TH staging issues.
+deriveSafeCopySimple 0 'base ''AddrSpendingData
+deriveSafeCopySimple 0 'base ''AddrType -- ☃
```

Comment:
Not an actual issue, but do we know what ☃ snowman means?

Reply:
Nope, Just left it there for kicks :)